### PR TITLE
Created a branch called remoted-branch

### PR DESCRIPTION
**Pull request by moise-mulungu**
From this branch I added the following contents:
-A section called footer
- Used the form with a value "action" and a method with "post"
- Applied hover and active CSS properties.
- Updated my "contact me" page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -222,7 +222,7 @@ <h2 class="view">I'm always<br>
             <hr>
             <input type="text" class="input" placeholder="Email address">
             <hr>
-            <input type="text" class="box" placeholder="Enter text here">
+            <textarea class="box" placeholder="Enter text here"></textarea>
```
Comment (medaminedev66) [inline index.html]: - [ ] Following the project requirements, All input fields should have the required attribute to activate the validation on submit. Kindly add it 😃.
- [ ] Message input also should have `maxlength="500"` attribute.

File: index.html [hunk 2]
```diff
@@ -222,7 +222,7 @@ <h2 class="view">I'm always<br>
             <hr>
             <input type="text" class="input" placeholder="Email address">
```
Comment (medaminedev66) [inline index.html]: - [ ] Following the project requirements, the email input's `type` should equal `email`

File: index.html [hunk 3]
```diff
@@ -222,7 +222,7 @@ <h2 class="view">I'm always<br>
             <hr>
```
Comment (medaminedev66) [inline index.html:220]: - [ ] Please make sure that the form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).
* `action` value should be similar to `https://formspree.io/f/{form_id}`
* `method` value equals `POST`
- [ ] You should also consider that the name input should have `maxlength="30"` attribute.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @moise-mulungu , :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_See the inline comments below for more required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -217,14 +217,14 @@ <h2 class="view">I'm always<br>
...
-            <input type="text" class="input" placeholder="Full name">
+        <form action="https://formspree.io/f/xzbodvoe" method="post">
+            <input type="text" class="input" placeholder="Full name" required maxlength="30">
             <hr>
-            <input type="text" class="input" placeholder="Email address">
+            <input type="email" class="input" placeholder="Email address" required>
             <hr>
-            <input type="text" class="box" placeholder="Enter text here">
+            <textarea class="box" placeholder="Enter text here" required maxlength="500"></textarea>
+            <button type="button" class="touch">Get in touch</button>
```
Comment (medaminedev66) [inline index.html]: - [ ] You have done a great job by implementing the requested changes. But we still cannot submit the form. Kindly change the type of the button from `button` to `submit`.
- [ ] Please before requesting another review, make sure your form is submitted successfully. You should see this message below after submitting the form:
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @moise-mulungu, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @moise-mulungu** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 

- The form is submitted successfully with valid input values :rocket: 


Cheers and Happy coding!👏👏👏💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**